src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -207,18 +232,14 @@ private function extractFromConstructor(string $class, string $property): ?array
             return null;
         }
 
-        $constructor = $reflectionClass->getConstructor();
-
-        if (!$constructor) {
+        if (!$constructor = $reflectionClass->getConstructor()) {
             return null;
         }
 
         foreach ($constructor->getParameters() as $parameter) {
-            if ($property !== $parameter->name) {
-                continue;
+            if ($property === $parameter->name) {
+                return array($this->extractFromReflectionType($parameter->getType()));
             }
```

Comment:
what's the point of this change ? I know it does the same, but with a different style, but then ?

Reply:
Removing 2 lines and making the code more explicit, but yes it's not mandatory.